src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,481 @@
+GRANT UPDATE ON "SDT_OWNER"."SERVICE_REQUESTS" TO "SDT_USER";
+GRANT DELETE ON "SDT_OWNER"."TARGET_APPLICATIONS" TO "SDT_USER";
+GRANT INSERT ON "SDT_OWNER"."TARGET_APPLICATIONS" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."TARGET_APPLICATIONS" TO "SDT_USER";
+GRANT UPDATE ON "SDT_OWNER"."TARGET_APPLICATIONS" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."BULK_CUST_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."BULK_CUST_APP_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."BULK_SUB_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."ERR_LOG_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."ERR_MESG_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."GLB_PAR_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."IND_REQ_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SER_ROU_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SER_TYP_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."TAR_APP_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SDT_REF_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SRV_REQ_SEQ" TO "SDT_USER";
+-----------------
+--- System Grants
+-----------------
+--GRANT UNLIMITED TABLESPACE TO "SDT_OWNER";
+--GRANT CREATE SYNONYM TO "SDT_USER";
+--GRANT CREATE SYNONYM TO "SDT_BATCH_USER";
+-----------------
+--- Sys Grants
+-----------------
+--GRANT EXECUTE ON "SYS"."UTL_FILE" TO "SDT_OWNER";
+
+------------------------------------------------
+-- Create Views (not Synonyms!) for SDT_USER
+------------------------------------------------
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUSTOMERS" AS SELECT * FROM "SDT_OWNER"."BULK_CUSTOMERS";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUSTOMER_APPLICATIONS" AS SELECT * FROM "SDT_OWNER"."BULK_CUSTOMER_APPLICATIONS";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUST_APP_SEQ" AS SELECT * FROM "SDT_OWNER"."BULK_CUST_APP_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUST_SEQ" AS SELECT * FROM "SDT_OWNER"."BULK_CUST_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_SUBMISSIONS" AS SELECT * FROM "SDT_OWNER"."BULK_SUBMISSIONS";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_SUB_SEQ" AS SELECT * FROM "SDT_OWNER"."BULK_SUB_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."ERROR_LOGS" AS SELECT * FROM "SDT_OWNER"."ERROR_LOGS";
+CREATE OR REPLACE VIEW "SDT_USER"."ERROR_MESSAGES" AS SELECT * FROM "SDT_OWNER"."ERROR_MESSAGES";
+CREATE OR REPLACE VIEW "SDT_USER"."ERR_LOG_SEQ" AS SELECT * FROM "SDT_OWNER"."ERR_LOG_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."ERR_MESG_SEQ" AS SELECT * FROM "SDT_OWNER"."ERR_MESG_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."GLB_PAR_SEQ" AS SELECT * FROM "SDT_OWNER"."GLB_PAR_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."GLOBAL_PARAMETERS" AS SELECT * FROM "SDT_OWNER"."GLOBAL_PARAMETERS";
+CREATE OR REPLACE VIEW "SDT_USER"."INDIVIDUAL_REQUESTS" AS SELECT * FROM "SDT_OWNER"."INDIVIDUAL_REQUESTS";
+CREATE OR REPLACE VIEW "SDT_USER"."IND_REQ_SEQ" AS SELECT * FROM "SDT_OWNER"."IND_REQ_SEQ";
+--CREATE OR REPLACE VIEW "SDT_USER"."MESSAGE_LOGS" AS SELECT * FROM "SDT_OWNER"."MESSAGE_LOGS";
```

Comment:
message_logs may no longer exist.  Can remove this if that's correct.

Reply:
Can't see any other references to MESSAGE_LOGS in the code so it appears this is redundant and can be removed.